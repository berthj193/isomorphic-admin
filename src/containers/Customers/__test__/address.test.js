import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Address from '../address';

describe('<Address />', () => {
  it('should render properly', () => {
    const props = {
      line_one: 'lorem-ipsum',
      line_two: 'dolor-sit-amet',
      postcode: 'P0stC0d3',
    };
    expect(toJson(shallow(<Address {...props} />))).toMatchSnapshot();
  });

  it('should skip second line of address', () => {
    const props = {
      line_one: 'lorem-ipsum',
      postcode: 'P0stC0d3',
    };
    expect(toJson(shallow(<Address {...props} />))).toMatchSnapshot();
  });
});
