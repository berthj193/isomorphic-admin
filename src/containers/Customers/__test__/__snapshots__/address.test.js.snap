// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Address /> should render properly 1`] = `
<div
  className="customer-address"
>
  <p>
    lorem-ipsum
  </p>
  <p>
    dolor-sit-amet
  </p>
  <p>
    P0stC0d3
  </p>
</div>
`;

exports[`<Address /> should skip second line of address 1`] = `
<div
  className="customer-address"
>
  <p>
    lorem-ipsum
  </p>
  <p>
    P0stC0d3
  </p>
</div>
`;
